src/test/java/org/eolang/jeo/representation/directives/OpcodeNameTest.java

```diff
@@ -53,6 +54,22 @@ void checksOpcodeNames(final int actual, final String expected) {
         );
     }
 
+    @Test
+    void returnsOpcode() {
+        final int expected = 0;
+        final int code = new OpcodeName(expected, OpcodeNameTest.COUNTER)
+            .code();
+        MatcherAssert.assertThat(
+            String.format(
+                "Opcode code: %s does not matches with expected: %s",
```

Comment:
@h1alexbel NIT: it's better to use `%d` for numbers.  [Docs.](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Formatter.html)